fem/bilininteg.cpp

```diff
@@ -3085,6 +3085,8 @@ void VectorDiffusionIntegrator::AssembleElementVector(
    // NOTE: DenseMatrix is in column-major order. This is consistent with
    // vectors ordered byNODES. In the resulting DenseMatrix, each column
    // corresponds to a particular vdim.
+   elfun.HostRead();
+   elvect.HostReadWrite();
```

Comment:
I this really ever moved to GPU? In what case do we need that?

Reply:
moved outside of `bilininteg` in [f62a2f1](https://github.com/mfem/mfem/pull/4675/commits/f62a2f100fbe31f2175f59fa1099cb151ba3ad3d)